components/prize_pool/commons/prize_pool.lua

```diff
@@ -415,9 +417,30 @@ function PrizePool:_buildRows()
 				row:addCell(cell)
 			end
 
-			-- TODO: Proper Support for Party Types
-			local opponentDisplay = OpponentDisplay.InlineOpponent{opponent = opponent.opponentData}
-			row:addCell(TableCell{content = {opponentDisplay}, css = {['justify-content'] = 'start'}})
+			local opponentDisplay = tostring(OpponentDisplay.InlineOpponent{opponent = opponent.opponentData})
+			local opponentCss = {['justify-content'] = 'start'}
+
+			if self:_hasPartyType() then
+				if Opponent.typeIsParty(opponent.opponentData.type) then
+					row:addCell(TableCell{content = {opponentDisplay}, css = opponentCss})
+				else
+					row:addCell(TableCell{content = {DASH}})
+				end
+			end
+
+			if Opponent.typeIsParty(opponent.opponentData.type) then
+				-- Really don't like this
+				-- Move to Opponent Display?
```

Comment:
Move which part exactly?

Reply:
Have team content for a player be generated by a function in OpponentDisplay, allows for skipping the opponent.players[1].team and all the checks if it exists. Would allow re-usage of code as well that core of display.

Reply:
Dropped one of the ifs. Slightly better now.

Will look in a future PR. Removed the comment as planning on changing how this is handled in the future (see [hjp's mockup](https://user-images.githubusercontent.com/75081997/175276209-f831cc54-38cf-4d33-adbc-12617aa054c6.png))